i18n/i18n.go

```diff
@@ -83,10 +83,7 @@ func bindTextDomain(domain, dir string) {
 
 func setLocale(loc string) {
 	if loc == "" {
-		loc = os.Getenv("LC_MESSAGES")
-		if loc == "" {
-			loc = os.Getenv("LANG")
-		}
+		loc = CurrentLocale()
 	}
 	// de_DE.UTF-8, de_DE@euro all need to get simplified
 	loc = strings.Split(loc, "@")[0]
```

Comment:
WIth the code changes, this is already done in CurrentLocale()

Reply:
I refactored this to be in a shared helper so we don't run it multiple times